libminifi/src/io/FileStream.cpp

```diff
@@ -113,34 +113,33 @@ void FileStream::seek(size_t offset) {
     logging::LOG_ERROR(logger_) << SEEK_ERROR_MSG << SEEKP_CALL_ERROR_MSG;
 }
 
-int FileStream::write(const uint8_t *value, int size) {
-  gsl_Expects(size >= 0);
+size_t FileStream::write(const uint8_t *value, size_t size) {
   if (size == 0) {
     return 0;
   }
   if (!IsNullOrEmpty(value)) {
```

Comment:
we could refactor all these (I counted 3) instances to use early return as you have done it in `SecureDescriptorStream.cpp`

Reply:
done in 5c6ba7b